src/inversify.config.ts

```diff
@@ -0,0 +1,15 @@
+import { Container } from 'inversify';
+import TaskService from './modules/task/task.service';
+import UserService from './modules/users/user.service';
+import RedisService from './services/redis/redis.service';
+import { RedisServiceInterface } from './services/redis/redis.types';
+import { TYPES } from './types/types';
+
+const container = new Container();
+
+// Регистрация сервисов
+container.bind<TaskService>(TYPES.TaskService).to(TaskService).inSingletonScope();
+container.bind<UserService>(TYPES.UserService).to(UserService).inSingletonScope();
+container.bind<RedisServiceInterface>(TYPES.RedisService).to(RedisService).inSingletonScope();
```

Comment:
```suggestion
container.bind(TaskService).toSelf().inSingletonScope();
```
И все остальные зависимости так же